src/test/mixxxtest.cpp

```diff
@@ -43,18 +38,24 @@ MixxxTest::ApplicationScope::ApplicationScope(int& argc, char** argv) {
     // in the same thread during tests to prevent test failures
     // due to timing issues.
     disableConcurrentGuessingOfTrackCoverInfoDuringTests();
+
+    DEBUG_ASSERT(s_pApplication.isNull());
+    s_pApplication.reset(new MixxxApplication(argc, argv));
+
+    VERIFY_OR_DEBUG_ASSERT(SoundSourceProxy::registerSoundSourceProviders()) {
+    }
```

Comment:
what's the purpose of VERIFY_OR_DEBUG_ASSERT without a body? Wouldn't a regular DEBUG_ASSERT suffice?